src/main/java/com/hartwig/platinum/ApiRerun.java

```diff
@@ -28,47 +27,50 @@ public class ApiRerun {
     private final String bucket;
     private final String version;
 
-    public ApiRerun(final RunApi runApi, final SetApi setApi, final SampleApi sampleApi, final String bucket, final String version) {
+    public ApiRerun(final RunApi runApi, final SetApi setApi, final SampleApi sampleApi, final String bucket,
```

Comment:
I think you're using the wrong intellij code scheme settings here. It's giving a lot of unnecessary diffs. I think for platinum we previsously used the pipeline5 code scheme? At least, when I apply that code scheme a lot of the diff disappears. We discussed consolidating a few weeks ago. I think it's still good to do, but maybe as a single commit for all projects at once some time in the future.